PWGDQ/TableProducer/tableMaker.cxx

```diff
@@ -349,7 +358,11 @@ struct TableMaker {
             }
           }
         }
+
         trackFilteringTag |= (uint64_t(trackTempFilterMap) << 7); // BIT7-14:  user track filters
+        if constexpr (static_cast<bool>(TTrackFillMap & VarManager::ObjTypes::DalitzBits)) {
+          trackFilteringTag |= (uint64_t(track.dalitzBits()) << 15); // BIT15-...: Dalitz
+        }
```

Comment:
I suggest that we inject the Dalitz bits right after the V0 bits. The user track filters should be at the end.
We can probably reserve 8 bits just for the Dalitz. The user cuts might be many more after we discuss the cuts in the DQ for the skimming strategy, so they would fill up the space up to 64. You need to do this change also in the tableMakerMC and in the VarManager, where you have to correct for the bit position in the variables you defined.